python/doc/examples/reliability_sensitivity/sensitivity_sobol.ipynb

```diff
@@ -342,10 +342,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "- We see that the variable $X_1$, with a total Sobol' indice close to 0.6, is the most significant variable, taking into account its effet by itself or with its interactions with other variables. The first order indice is close to 0.3, which implies that, for the interactions (and only the interactions) produce almost 0.3 of the total variance. \n",
-    "- The variable $X_2$ has the highest first order indice, but has no interactions since the total order indice is close to the first order indice. Its total Sobol' indice is approximately equal to 0.4. \n",
-    "- The variable $X_3$ has a first order indice close to zero. However, it has an impact to the total variance thanks to its interactions with $X_1$. \n",
-    "- We see that, because of the sampling, the variability of the Sobol' indices estimates is quite large, even with a relatively large sample size. Moreover, since the exact first order Sobol' indice for $X_3$ is zero, its estimate has 50% chance of being nonpositive. "
+    "- We see that the variable $X_1$, with a total Sobol' index close to 0.6, is the most significant variable, taking into account both its direct effect and its interactions with other variables. Its first order index is close to 0.3, which implies that its interactions alone produce almost 30% of the total variance. \n",
```

Comment:
```suggestion
    "- We see that the variable $X_1$, with a total Sobol' index close to 0.6, is the most significant variable, taking into account both its direct effect and its interactions with other variables. Its first order index is close to 0.3, which implies that its interactions alone produce almost 30% (0.6 - 0.3) of the total variance. \n",
```
I feel some clarification is necessary, otherwise a distracted reader might think that the first order index measures the influence of the  interactions...

Reply:
Fixed.